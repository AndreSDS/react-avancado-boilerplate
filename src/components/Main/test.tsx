import { render, screen } from '@testing-library/react'

import { Main } from '.'

const main = render(<Main />)

describe('<Main />', () => {
  it('should render a heading', () => {
    const { container } = main

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
