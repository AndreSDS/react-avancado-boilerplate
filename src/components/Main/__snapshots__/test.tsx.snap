// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render a heading 1`] = `
<main
  class="styles__Wrapper-sc-dzx1ao-0 jTzoJi"
>
  <img
    alt="imagem de um átomo e um texto escrito react avançado"
    class="styles__Logo-sc-dzx1ao-1 dwiHcF"
    src="/img/logo.svg"
  />
  <h1
    class="styles__Title-sc-dzx1ao-2 kYhHsR"
  >
    React Avançado
  </h1>
  <h2
    class="styles__Description-sc-dzx1ao-3 cWfoFI"
  >
    TypeScript, ReactJS, NextJs e Styled Components
  </h2>
  <img
    alt="um desenvolvedor sentado na cadeira e em frente ao computador"
    class="styles__Illustration-sc-dzx1ao-4 dbHwHR"
    src="/img/hero-illustration.svg"
  />
</main>
`;
